data/applications/nvidia.yaml

```diff
@@ -0,0 +1,18 @@
+metadata:
+  name: nvidia
+  annotations:
+    opendatahub.io/categories: 'GPU'
+spec:
+  displayName: NVIDIA GPU Add-on
+  provider: Nvidia
+  description: |-
+    NVIDIA GPU Add-on prepares OpenShift to run GPU-accelerated workloads on a single node or on multiple nodes.
+  kfdefApplications: []
+  route: ''
+  img: images/nvidia.svg
+  category: Self-managed
+  support: third party support
+  csvName: gpu-operator-certified
+  docsLink: https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/contents.html
+  quickStart: gpu-quickstart
+  getStartedLink: https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science/1/html/managing_users_and_user_resources/enabling-gpu-support-in-openshift-data-science_user-mgmt
```

Comment:
I believe you need an enable section to allow a user to enable this add-on in the explore section.

Here's one without validation:

https://github.com/red-hat-data-services/odh-dashboard/blob/8680a17ee2f4d1d92b1edb950e97e6b7871b5e0b/data/applications/rhosak.yaml#L18

Here's an example where validation is needed. 
https://github.com/red-hat-data-services/odh-dashboard/blob/8680a17ee2f4d1d92b1edb950e97e6b7871b5e0b/data/applications/anaconda-ce.yaml#L16